### PR TITLE
Fixes crashing tests issue.

### PR DESCRIPTION
**Pull request by PatrickPenguinTurtle**
UI tests were not calling NativesLoader properly before starting, and thus the first time any Opencv class was referenced it would fail. 

Now GripCoreModule ensures that NativesLoader is always called before setup of the CoreModule which contains most of the Opencv references.

All tests should work now.

### CODE REVIEW

File: core/src/main/java/edu/wpi/grip/core/operations/opencv/NewPointOperation.java [hunk 1]
```diff
@@ -42,6 +42,8 @@ public NewPointOperation(InputSocket.Factory inputSocketFactory, OutputSocket.Fa
     this.ySocket = inputSocketFactory.create(yHint);
 
     this.outputSocket = outputSocketFactory.create(outputHint);
+    
+    this.perform();//So initial value is correct.
```
Comment (SamCarlberg) [inline core/src/main/java/edu/wpi/grip/core/operations/opencv/NewPointOperation.java]: **DO NOT DO THIS**

Use a builder for the socket hint to define a default value.

File: core/src/main/java/edu/wpi/grip/core/operations/opencv/NewSizeOperation.java [hunk 2]
```diff
@@ -41,6 +41,8 @@ public NewSizeOperation(InputSocket.Factory inputSocketFactory,
     this.heightSocket = inputSocketFactory.create(heightHint);
 
     this.outputSocket = outputSocketFactory.create(outputHint);
+    
+    this.perform();//So initial value is correct.
```
Comment (SamCarlberg) [inline core/src/main/java/edu/wpi/grip/core/operations/opencv/NewSizeOperation.java]: **DO NOT DO THIS**

Use a builder for the socket hint to define a default value.

Comment (PatrickPenguinTurtle) [inline core/src/main/java/edu/wpi/grip/core/operations/opencv/NewSizeOperation.java]: Working on this now. Is it better to add a parameter to the existing create size socket hint or just create a new method?

Comment (SamCarlberg) [inline core/src/main/java/edu/wpi/grip/core/operations/opencv/NewSizeOperation.java]: Use the pre-existing `SocketHint.Builder` class, that has everything you need

Comment (PatrickPenguinTurtle) [inline core/src/main/java/edu/wpi/grip/core/operations/opencv/NewSizeOperation.java]: As far as I can tell SocketHint.Builder is package private scope. So I would need to add a method to SocketHints.Outputs to call SocketHint.Builder. Right?

Comment (PatrickPenguinTurtle) [inline core/src/main/java/edu/wpi/grip/core/operations/opencv/NewSizeOperation.java]: Ignore previous comment. Was missing the word "new".

**Comment by SamCarlberg** [comment]
@PatrickPenguinTurtle Are you going to continue with this, or should I close this PR and pick up where you left off?

**Pull request closed without merging**